source/style-guide/style/placeholder-variable-text.txt

```diff
@@ -1,147 +1,182 @@
-       with spaces or symbols. To distinguish the words in the
-       placeholder, capitalize the first letter of each word after the
-       first word (called camelCase). Don't capitalize the first word.
+       To show a multiple-word placeholder:
+
+       - Don't separate the words with spaces or symbols
+       - Capitalize the first letter of each word after the first word
+         (called :wikipedia:`camelCase </Camel_case>`)
+       - Never capitalize the first word
 
        .. note::
-          Use lowercase and camelCase unless you have to follow the
-          conventions of the programming language. For example, you
-          might need to use underscores (account_ID) or all capitals
-          (ACCOUNT_ID).
+          Use lowercase and camelCase unless you need to follow the
+          conventions of the programming language.
+
+     - *password*, *eventTypeName*, *apiKey*, *hostId*
 
-     - *password* *serverName* *apiKey* *tenantId*
+   * - Use one or more whole words to represent a placeholder.
 
-   * - In general, use one or more whole words to represent a
-       placeholder. Don't sacrifice clarity for brevity. Create
-       placeholders that are descriptive and meaningful.
+       - Don't sacrifice clarity for brevity.
+       - Create descriptive and meaningful placeholders.
 
-     - *device* (instead of *dev*)
+     - *device*; not *dev*
 
-       *installationDirectory* (instead of *installDir*)
+       *installationDirectory*; not *installDir*
 
-       *mode* (instead of *########*)
+       *mode*; not *########*
 
-When explaining a placeholder, use the following guidelines.
+Placeholder Explanation Guidelines
+----------------------------------
+
+When explaining a placeholder, use the following guidelines:
 
 .. list-table::
    :widths: 40 60
    :header-rows: 1
 
-   * - Guidelines
+   * - Guideline
 
      - Example
 
-   * - Avoid stand-alone clauses that begin with *where*. Instead, use
-       a sentence.
-
-     - *Use:*
-
-       **https://dfw.bigdata.api.MongoDBcloud.com/v1.0/yourAccountId/**
-
-       *yourAccountId* is your actual account number, which is returned
-       as part of the authentication service response.
-
-       *Avoid:*
+   * - Use a sentence to describe the placeholder parameter.
 
-       **https://dfw.bigdata.api.MongoDBcloud.com/v1.0/yourAccountId/**
+     - .. code-block:: console
 
-       where *yourAccountId* is your actual account number, which is
-       returned as part of the authentication service response.
+          mongod --port <portNumber> \
+                 --dbpath <tempDatabasePath> \
+                 --setParameter ttlMonitorEnabled=false
 
-   * - If you need to explain two or more placeholders, use an
-       unordered list.
+       *port* is the unique number for the ``mongod`` service. The
+       default value is ``27017``.
 
-     - From a supported web browser, type the following URL:
+   * - Avoid standalone clauses that begin with *where*.
```

Comment:
Nitpick: I wonder if it would be helpful to separate the group of "do this" from the "avoid doing this" items. Alternatively, maybe "Avoid" can be bolded here to emphasize the signal word.